src/libs/actions/App.js

```diff
@@ -105,6 +105,42 @@ function setLocaleAndNavigate(locale) {
     Navigation.navigate(ROUTES.SETTINGS_PREFERENCES);
 }
 
+/**
+ * @param {String} colorTheme
+ */
+// eslint-disable-next-line rulesdir/prefer-early-return
+function setColorTheme(colorTheme) {
```

Comment:
For this PR, we can just merge and test the page and move this section into a separate PR for later. I'd copy this into your local test steps in the body of the description